File: QuizHub.Domain/Models/UnidentifiedQuestionWithNoImage.cs
using QuizHub.Domain.Enums;

namespace QuizHub.Domain.Models;

public class UnidentifiedQuestionWithNoImage
{
    public string Content { get; set; } = string.Empty;
    public QuestionType QuestionType { get; set; }
    public List<UnidentifiedAnswerWithNoImage> Answers { get; set; } = [];
}

File: QuizHub/src/app/manager/models/unidentifiedQuestionWithNoImage.ts
import { QuestionType } from '../../common/enums/questionType';
import { UnidentifiedAnswerWithNoImage } from './unidentifiedAnswerWithNoImage';

export type UnidentifiedQuestionWithNoImage = {
  content: string;
  answers: UnidentifiedAnswerWithNoImage[];
  questionType: QuestionType;
};
